### PR TITLE
CRUD of experiments and experiments API

### PR DESCRIPTION
**Pull request by sufka861**

### CODE REVIEW

File: controller/experiment.controller.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    if (!req.query.year && req.query.month) throw new PropertyNotFound("year and month");
+    const year = req.query.year;
+    const month = req.query.month;
+    const result = await experimentRepository.findByDate(year, month);
+    if (!result) throw new ServerUnableError("getExperimentsByDate")
+    res.status(200).json({result});
+}
+
+const createExperiments = async (req, res) => {
+    const result = await experimentRepository.create(req.body)
```
Comment (AtomicManic) [inline controller/experiment.controller.js:57]: missing body validation

File: db/MongoStorage.js [hunk 2]
```diff
@@ -1,43 +1,74 @@
...
+    findGroup(key, value, key2, value2) {
+        const obj = {};
+        obj[key] = value;
+        if (key2 && value2) {
+            obj[key2] = value2;
+        }
+        return this.Model.find(obj);
+    }
+
+    findByAccount(key, value) {
```
Comment (AtomicManic) [inline db/MongoStorage.js:39]: change to find by attribute

File: repositories/experiment.repository.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    update(id, experiment) {
+        return this.storage.update(id, experiment);
+    }
+
+    delete(id) {
+        return this.storage.delete(id);
+    }
+
+    findGroup(key, value, key2, value2) {
```
Comment (AtomicManic) [inline repositories/experiment.repository.js:29]: here you can change this to find by dates

**Review by AtomicManic — COMMENTED** [review]
Great Code!!!!

**Review by AtomicManic — APPROVED** [review]
Great Code!!!!
Minor changes and we are good to go!

**Comment by sufka861** [comment]
fixed everything :)

**Pull request merged**